server/athenian/api/controllers/miners/github/commit.py

```diff
@@ -200,6 +200,7 @@ async def fetch_repository_commits(repos: Dict[str, DAG],
     missed_counter = 0
     repo_heads = {}
     sha_col, id_col, dt_col, repo_col = columns
+    branches = branches[branches[id_col].notnull()]
```

Comment:
Well, this is impossible because of https://github.com/athenianco/athenian-api/blob/7fff5199f65978416895d2e9b3594b3810d9c0a7/server/athenian/api/controllers/miners/github/branches.py#L127 I know that releases may pretend as branches, too, but this is still a huge violation of the input constraints. In other words, if this happens and some branches appear without commits, we must assert and panic instead of silently filtering them away.